physics/module_sf_ruclsm.F90

```diff
@@ -879,36 +911,48 @@ SUBROUTINE LSMRUC(                                           &
               LMAVAIL(I,J)=max(0.00001,min(1.,soilm1d(1)/(ref-qmin)))
 
     IF (debug_print ) THEN
-   print *,'LAND, i,j,tso1d,soilm1d,PATM,TABS,QVATM,QCATM,RHO',  &
-                  i,j,tso1d,soilm1d,PATM,TABS,QVATM,QCATM,RHO
-   print *,'CONFLX =',CONFLX 
-   print *,'SMFRKEEP,KEEPFR   ',SMFRKEEP,KEEPFR
+      if (abs(xlat-testptlat).lt.0.2 .and.                           &
+          abs(xlon-testptlon).lt.0.2)then
+        print *,'LAND, i,j,tso1d,soilm1d,PATM,TABS,QVATM,QCATM,RHO',  &
+                       i,j,tso1d,soilm1d,PATM,TABS,QVATM,QCATM,RHO
+        print *,'CONFLX =',CONFLX 
+        print *,'SMFRKEEP,KEEPFR   ',SMFRKEEP,KEEPFR
+      endif
     ENDIF
 
         smtotold(i,j)=0.
-      do k=1,nzs-1
+
+      !do k=1,nzs-1
+      do k=1,nroot
         smtotold(i,j)=smtotold(i,j)+(qmin+soilm1d(k))*             &
                     (zshalf(k+1)-zshalf(k))
       enddo
 
-        smtotold(i,j)=smtotold(i,j)+(qmin+soilm1d(nzs))*           &
-                    (zsmain(nzs)-zshalf(nzs))
+        !smtotold(i,j)=smtotold(i,j)+(qmin+soilm1d(nzs))*           &
```

Comment:
No need to comment code with github history.

Reply:
Sometimes I keep commented out lines for my memory, but you are right, it does make the code look messy. Removed the commented lines.